resources/js/components/View.tsx

```diff
@@ -0,0 +1,17 @@
+import { BrowserRouter, Route, Routes } from "react-router-dom";
+
+import Login from "./Login";
+import Signup from "./Signup";
+
+function View() {
+    return (
+        <BrowserRouter>
+            <Routes>
```

Comment:
```suggestion
import Routes from './routes';

            <Router>
               <Routes />
           </Router>
```
What I meant is like this. Just the routes itself, not the the whole View.

Reply:
Gotcha, modified.

![image](https://user-images.githubusercontent.com/105616953/173047426-f18a1723-a0ef-4b52-807a-e76cbd48a2ab.png)